src/view/upcastwriter.js

```diff
@@ -172,6 +172,27 @@ export default class UpcastWriter {
 		return false;
 	}
 
+	/**
+	 * Removes given element from view structure and places its children in its position.
+	 *
+	 * @param {module:engine/view/element~Element} element Element which will be unwrapped
+	 * @returns {Booolean} Whether element was successfully unwrapped.
+	 */
+	unwrapElement( element ) {
+		const parent = element.parent;
+
+		if ( parent ) {
+			const index = parent.getChildIndex( element );
+
+			this.remove( element );
+			this.insertChild( index, element.getChildren(), parent );
+
+			return true;
```

Comment:
The method should not return true/false - we don't need that.

Check the model's `writer.unwrap()` how we handle this: https://github.com/ckeditor/ckeditor5-engine/blob/ad9159ca794096cbed1ab44180df0c74181d2f31/src/model/writer.js#L851-L861